### PR TITLE
Pulled in v0.10 and fixed merge conflicts

### PR DESCRIPTION
**Pull request by sothychan**

### CODE REVIEW

**Comment by sothychan** [comment]
Actually, finding some issues during build time.  Going to fix.

File: plugin/modelgen/models_test.go [hunk 1]
```diff
@@ -14,8 +14,14 @@ import (
 func TestModelGeneration(t *testing.T) {
 	cfg, err := config.LoadConfig("testdata/gqlgen.yml")
 	require.NoError(t, err)
+<<<<<<< HEAD
```
Comment (marwan-at-work) [inline plugin/modelgen/models_test.go]: conflict metadata

Comment (sothychan) [reply plugin/modelgen/models_test.go]: Ah.  Thanks for finding that.  Will fix.

Comment (sothychan) [reply plugin/modelgen/models_test.go]: Fix pushed up.  All tests passed.

**Pull request merged**